firefox-ios/Client/Frontend/Toolbar+URLBar/ShareButton.swift

```diff
@@ -14,10 +14,10 @@ class ShareButton: UIButton {
     var disabledTintColor: UIColor!
 
     override open var isHighlighted: Bool {
-        didSet {
-            self.tintColor = isHighlighted ? selectedTintColor : unselectedTintColor
+            didSet {
+                self.tintColor = isHighlighted ? selectedTintColor : unselectedTintColor
+            }
```

Comment:
It seems this is possibly too much indented?

Reply:
yes, missed it. Updated :)